Clarified experimental nature of crux in its own section, which state… (#288)
redbadger/crux: docs/src/overview.md

# Overview

> Crux is an approach to building cross-platform applications

Reviewer: ```suggestion
Crux is a framework for building cross-platform applications
```

- Separate the behavior from the look and feel and interaction design
- Use the native UI tool kits to create user experience that is the best fit for
  a given platform

## Path to 1.0

> Crux is used in production apps today, at least one quite large one. In that sense
> we consider it production ready. However, to truly be at 1.0 with a more stable
> API, we will take the following steps:

Reviewer: ```suggestion
Crux is used in production apps today, and we consider it production ready. However, we still have a number of things to work on to call it 1.0, with a stable API, and other things one would expect from a mature framework. 

Below is a list of some of the things we know we want to do before 1.0:
```

* Improved documentation, code examples, and example apps for newcomers
* Improved onboarding experience, with less boilerplate code that end users have
  to deal with

> * Better FFI code generation to enable support for more Shells (e.g. .NET,
>   Flutter) which will also enable Desktop apps in Windows

Reviewer: ```suggestion
* Better FFI code generation to enable support for more languages (e.g. C#, Dart, even C++...) 
  and in trurn more Shells (e.g. .NET, Flutter) which will also enable Desktop apps for Windows
```

> * Revised capabilities and effects to allow
>   for [app composition](./guide/composing.md) in larger apps

Reviewer: ```suggestion
* Revised capabilities and effects to allow for better, more natural
  [app composition](./guide/composing.md) in larger apps, for composing capabilities, 
  and generally for a more ergonomic effect API overall
```

> Until then, we hope you will work with us on the rough edges, and factor in
> frequent API updates as we evolve.

Reviewer: ```suggestion
Until then, we hope you will work with us on the rough edges, and adapt to the necessary
API updates as we evolve. We strive to minimise the impact of changes as much as we can, but before 1.0, some breaking changes will be unavoidable. 
```
